src/main/scala/strawman/collection/immutable/BitSet.scala

```diff
@@ -27,6 +27,7 @@ sealed abstract class BitSet
   def empty: BitSet = BitSet.empty
 
   def iterableFactory = Set
+  def sortedIterableFactory = SortedSet
```

Comment:
Why can’t we tell that the factory of a `BitSet` collection is the `BitSet` object?